test/services/katello/candlepin/consumer_test.rb

```diff
@@ -62,6 +62,16 @@ def test_distribution_to_puppet_os
 
         assert_equal nil, Candlepin::Consumer.distribution_to_puppet_os('RedHot')
       end
+
+      def test_virtual_guests
+        guests = []
```

Comment:
Thanks for adding a test for this method!

What if we wrote it this way, to make it more readable, and reduce interaction with the database by using an existing 'fixture' instead of creating three new hosts?

```
      def test_virtual_guests                                                                                                                                                
              facet = katello_subscription_facets(:one)                                                                                                                      
              guests = [facet.host]                                                                                                                                          
              guest_uuids = [{'uuid' => facet.uuid}]                                                                                                                         
                                                                                                                                                                             
              Resources::Candlepin::Consumer.expects(:virtual_guests).once.returns(guest_uuids)                                                                              
                                                                                                                                                                             
              assert_equal guests, @consumer.virtual_guests                                                                                                                  
              assert_equal guests, @consumer.virtual_guests                                                                                                                  
      end
```

Reply:
Sure. will do.

Reply:
Done